.travis.yml

```diff
@@ -1,5 +1,14 @@
 language: python
 dist: xenial  # required for Python >= 3.7
 python: 3.7
-install: pip install flake8
-script: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+cache: pip
+install: pip install -r requirements.txt
+before_script:
+  - black --check . || true
+  - flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+script:
+  - mypy --ignore-missing-imports .
+  - pytest --doctest-modules ./ciphers ./other ./searches ./sorts ./strings
```

Comment:
Also it would be nice to add pytest-cov and generate coverage report?

Reply:
Sorry, I am not a fan of coverage tools.  We do not mandate tests.

Reply:
@sirex It has been a busy two weeks but now we have [132 pytests passing](https://travis-ci.org/TheAlgorithms/Python/jobs/561642563#L685) so I think we could add __pytest-cov__ to Travis CI as you suggested above.  Please consider submitting a pull request that modifies __.travis.yml__ so that it runs __pytest-cov__.